prow/plugins/trigger/generic-comment.go

```diff
@@ -121,6 +147,12 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 	if err != nil {
 		return err
 	}
+
+	addedComment, err := addHelpComment(c.GitHubClient, len(toTest), len(toSkip), gc.Body, org, repo, branch, pr.Number, presubmits, gc.HTMLURL, commentAuthor, c.Logger)
+	if addedComment {
```

Comment:
This does not feel right.

1. Do we ever want to continue to the `RunAndSkipJobs...` if `err != nil`?
2. The usual convention is that if `err != nil` then the other return values should not be depended on - this may surprise anyone who would fix something in `addHelpComment` without knowing the callsite depends on the first return value being correctly set even on an error.

Reply:
Thank you for the review. Fixed.